### DISCUSSION TITLE
SSL Configuration on https://current.geneontology.org

### DISCUSSION THREAD

**Discussion by rtaylorzfin** [geneontology/helpdesk#402] [General]
Hi,

Please let me know if this isn't the right place to post this. I noticed an issue on the SSL configuration for current.geneontology.org.

We are linking to http://current.geneontology.org/annotations/zfin.gaf.gz on our downloads page at ZFIN (https://zfin.org/downloads).  Unfortunately, most browsers give a security warning or ignore that link since it is coming from an https site and linking to a download on an http site.  We can change the link to https://current.geneontology.org/annotations/zfin.gaf.gz, but the browser warns of a certificate issue.  Would it be possible to fix the certificate issue so browsers don't give warnings on that domain?

Thanks,
Ryan Taylor

**Comment by kltm** [comment]
Ah, well that's an annoying new feature. Thank you for bringing this to our attention.
I'll note here that there is a distant relation to https://github.com/geneontology/go-site/issues/53 here, but in this case the cert is in place for the CDN, which is apparently misbehaving.
